gazelle/js/typescript/tsconfig_test.go

```diff
@@ -31,6 +47,45 @@ func assertExpand(t *testing.T, options *TsConfig, p string, expected ...string)
 	}
 }
 
+func TestBaseUrlRegex(t *testing.T) {
+	t.Parallel()
+
+	t.Run("valid strings", func(t *testing.T) {
+		t.Parallel()
```

Comment:
I don't think we use this in any of the other tests atm. Does it have to be in each test *and* above? I'd vote to just remove it for now, can think about it more in the future 🤷

Reply:
If the nesting cases with t.Run should also be parallel the t.Parallel must also be nested. Otherwise the wrapping test function will be parallel to other parallel tests and the nested ones are sequential inside the parallel wrapper func. But totally fair to remove it if not used otherwhere.